packages/opentelemetry-tracing/src/Span.ts

```diff
@@ -116,6 +116,14 @@ export class Span implements api.Span, ReadableSpan {
     return this;
   }
 
+  getAttribute(key: string): api.SpanAttributes {
+    return this.attributes[key];
+  }
+
+  getAttributes(): api.SpanAttributes {
```

Comment:
Not allowed by the specification. Also, since this returns the object reference, any changes made to it by the caller will be reflected in the attributes object, potentially creating an invalid state.

example:

```typescript
const circular = {};
circular.ref = circular;

const attributes = span.getAttributes();
attributes.someAttribute = circular;

// now an attribute has an unserializable value
```